frontend/scss/molecules/_m-editorial-block.scss

```diff
@@ -1,10 +1,14 @@
+
 .m-editorial-block {
     width: 100%;
     margin: 56px 0;
     display: flex;
     flex-direction: column;
     max-width: 1500px;
 
+    @import './m-block----quick-look';
```

Comment:
Rename all the `m-block` namings to `m-editorial-block----quick-look` so it's more immediately clear that they're related.